sysid-application/src/main/native/cpp/App.cpp

```diff
@@ -2,6 +2,8 @@
 // Open Source Software; you can modify and/or share it under the terms of
 // the WPILib BSD license file in the root directory of this project.
 
+#include <cstdio>
```

Comment:
Why is cstdio being included?

Reply:
Not sure, maybe because of the `stderr`?